core/src/main/java/io/grpc/internal/ServerImpl.java

```diff
@@ -609,34 +683,31 @@ public void cancelled(Context context) {
       ServerMethodDefinition<ReqT, RespT> interceptedDef = methodDef.withServerCallHandler(handler);
       ServerMethodDefinition<?, ?> wMethodDef = binlog == null
           ? interceptedDef : binlog.wrapMethodDefinition(interceptedDef);
-      return startWrappedCall(fullMethodName, wMethodDef, stream, headers, context, tag);
+      return wMethodDef;
+    }
+
+    private final class ServerCallParameters<ReqT, RespT> {
+      ServerCallImpl<ReqT, RespT> call;
+      ServerMethodDefinition<ReqT, RespT> methodDef;
```

Comment:
Use ServerCallHandler here to be more clear that the MethodDescriptor within the definition isn't used?